spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/interpreter/stages/TypeParser.scala

```diff
@@ -422,9 +422,12 @@ object TypeParser {
       origType match {
         case BinaryType => column
         case StringType =>
-          val validationIssues: Seq[ValidationIssue] = field.validate()
-          if (validationIssues.nonEmpty) {
-            throw new IllegalStateException(s"There are validation issues, cannot continue parsing binary field $field: $validationIssues")
+          val (validationErrors, validationWarnings) = field.validate().span(_.isInstanceOf[ValidationError])
```

Comment:
Sorry missed that in the previous review.
The validation is done before getting here in the Standardization flow. No point to doing it here again.